WikiWatershed/model-my-watershed
# AWS 2-5: Analyze Global Streams

## Overview

Adds analysis for global streams. The analysis is lazy, so it doesn't run until someone explicitly selects it from the dropdown. It is also cached when using WKAoIs.

The analysis doesn't include Agricultural Length, because we're not intersecting it with land use data. That is deferred until later.

The Contributing Area calculation needs feedback from the clients before finalizing.

This also adds Vagrant scripts to increase the disk size of the `services` VM to 128GB, to accommodate the new streams dataset.

Closes #3645 

### Demo

<img width="1556" alt="image" src="https://github.com/user-attachments/assets/df530d11-0e71-4e8a-a414-c8702e52318b">

https://github.com/user-attachments/assets/0e505c02-4d31-46c9-97db-323ecbcabf6a

## Testing Instructions

Testing this locally requires the addition of the global streams dataset, which is a multi-hour process. To make it easier, I recommend testing on staging, which has this branch deployed:

- Go to https://staging.modelmywatershed.org
- Select or draw an area of interest
- Select "Global Medium Resolution Streams" from the streams analysis tab's dropdown
